dwave/embedding/drawing.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# ================================================================================================
+
+from math import ceil, sqrt
+
+from dwave_networkx import chimera_graph, draw_chimera
+
+__all__ = ['draw_chimera_bqm']
+
+def draw_chimera_bqm(bqm, width=None, height=None):
+    """Draws a Chimera Graph representation of a Binary Quadratic Model.
+
+    If cell width and height not provided assumes square cell dimensions.
+    Throws an error if drawing onto a Chimera graph of the given dimensions fails.
+
+    Parameters
+    ----------
+    bqm : BinaryQuadraticModel
+        Should be equivalent to a Chimera graph or a subgraph of a Chimera graph.
```

Comment:
Should probably say that is should have matching nodes/edges. Maybe above say that it should match the graph generated by `dnx.chimera_graph`. Also it would be good to have an example

Reply:
Also the BQM should have integer variables/labels. And labels should correspond to the canonical Chimera, right? In other words, as @arcondello phrased it, it should be generated by our Chimera generator.